pkgs/applications/office/vnote/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, qt5, hicolor-icon-theme, makeDesktopItem }:
+
+let
+  description = "A note-taking application that knows programmers and Markdown better";
+  desktopItem = makeDesktopItem {
+    name = "VNote";
+    exec = "vnote";
+    icon = "vnote";
+    comment = description;
+    desktopName = "VNote";
+    categories = "Office";
+  };
+in stdenv.mkDerivation rec {
+  version = "2.5";
+  name = "vnote-v${version}";
+
+  src = fetchFromGitHub {
+    owner = "tamlok";
+    repo = "vnote";
+    fetchSubmodules = true;
+    rev = "8d780bd5898464139e252623ffaccd966e093ff1";
+    sha256 = "17nl4z1k24wfl18f6fxs2chsmxc2526ckn5pddi2ckirbiwqwm60";
+  };
+
+  installPhase = ''
```

Comment:
I'm going to guess if you use `cmake` over `qmake` you can probably get away with fewer manual instead steps, possibly just a `postInstall`.

Reply:
@kuznero What exactly was the issue you had that required a custom `installPhase`? I just commented out the `installPhase` you provided and the application built and ran as I would expect.

Reply:
Ha! Unexpected! Will test it without `installPhase`. Thanks!

Reply:
Fixed now. Please review